release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -72,6 +71,26 @@ Previously, secret information was used as the OAuth access token and OAuth auth
 If you are upgrading your cluster to {product-title} 4.6, old tokens from {product-title} 4.5 will still have the secret information exposed in the object name. By default, the expiration for tokens is 24 hours, but this setting can be changed by administrators. Sensitive data can still be exposed until all old tokens have either expired or have been deleted by an administrator.
 ====
 
+[id="ocp-4-6-scale"]
+=== Scale
+
+[id="ocp-4-6-scale-cluster-maximums"]
+==== Cluster maximums
+
+Updated guidance around
+xref:../scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc#planning-your-environment-according-to-object-maximums[Cluster
+maximums] for {product-title} {product-version} is now available.
+
+Use the link:https://access.redhat.com/labs/ocplimitscalculator/[{product-title}
+Limit Calculator] to estimate cluster limits for your environment.
+
+[id="ocp-4-6-scale-real-time-profile-added-to-node-tuning-operator"]
+==== Real-time profile added to the Node Tuning Operator
+
+Partial tuned real-time profile support became available in {product-title} 4.4.
```

Comment:
I'd suggest `s/tuned real-time profile/Tuned real-time profile/` other than that this looks good to me.  Thank you!